pkg/component/gardener/discoveryserver/secrets.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package discoveryserver
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/pkg/utils"
+	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
+)
+
+const serviceAccountIssuerConfigSecretName = "shoot-service-account-issuer"
```

Comment:
You can re-use the already defined constants
```suggestion
const serviceAccountIssuerConfigSecretName = v1beta1constants.GardenRoleShootServiceAccountIssuer
```